Catch the exception cast when git is not installed and just ignore it

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>

--- midori-0.1.3/wscript	2009-02-24 22:09:47.000000000 +0100
+++ midori-0.1.3/wscript.new	2009-02-24 22:09:39.000000000 +0100
@@ -27,9 +27,13 @@
 APPNAME = 'midori'
 VERSION = str (major) + '.' + str (minor) + '.' + str (micro)
 
-git = Utils.cmd_output (['git', 'rev-parse', '--short', 'HEAD'], silent=True)
-if git:
-    VERSION = (VERSION + '-' + git).strip ()
+
+try:
+    git = Utils.cmd_output (['git', 'rev-parse', '--short', 'HEAD'], silent=True)
+    if git:
+        VERSION = (VERSION + '-' + git).strip ()
+except:
+    pass
 
 srcdir = '.'
 blddir = '_build_'
